todo_app/app.py

```diff
@@ -14,31 +14,29 @@ def create_app():
     
     @app.route('/')
     def index():
-        response_json = trello_items.get_trello_cards()
+        mongo_items = mongo_items_Vtwo.get_mongo_items()
```

Comment:
`mongo_items_Vtwo` I suspect is your personal way of tracking a couple of different attempts, but it's not very helpful nomenclature for other users!